cmd/dsl/main.go

```diff
@@ -418,6 +488,13 @@ func main() {
 	var verbose bool
 	flag.BoolVar(&verbose, "v", false, "increase logging verbosity")
 	flag.Parse()
+
+	// validate flag-passed caps key
+	_, err := base64.StdEncoding.DecodeString(caps)
+	if err != nil {
+		fmt.Printf("Bail out! --caps %s was not a valid base64 sequence\n", caps)
+		return
+	}
```

Comment:
maybe also check the size of the decoded data is 32bytes?

the impls should also check that but why wait for that? (probably would also need to have verbose logging turned on to even see)